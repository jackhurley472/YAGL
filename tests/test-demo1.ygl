started();
void finished() {
	printString("Executed" + " Completely");
	printString("*******************");
}
String[3] foo;
String[3] t;
t[0] = "Make";
t[1] = "Some";
t[2] = "Noise";
int bar = 0;
while(bar < 3) {
	foo[bar] = t[bar];
	printString(foo[bar]);
	bar = bar + 1;
}
finished();
void started() {
	printString("******************");
	printString("Loading" + " ...");
}
