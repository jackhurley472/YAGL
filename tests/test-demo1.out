******************
Loading ...
Make
Some
Noise
Executed Completely
*******************
